lib/service/storage/elasticsearch.js

```diff
@@ -3393,6 +3417,12 @@ class ElasticSearch extends Service {
       }
     }
   }
+
+  async _getNumberOfNodes() {
```

Comment:
IMHO you could retrieve the number of nodes directly in `_getWaitForActiveShards` and avoid an extra function